Link error on sdv/ParkingAssist.lf

Math library link error happened while ParkingAssist.lf built on Linux as bellow.
```
/usr/bin/ld: CMakeFiles/ParkingAssist.dir/_sensors.c.o: undefined reference to symbol 'lround@@GLIBC_2.2.5'
/usr/bin/ld: /lib/x86_64-linux-gnu/libm.so.6: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
gmake[2]: *** [CMakeFiles/ParkingAssist.dir/build.make:249: ParkingAssist] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:100: CMakeFiles/ParkingAssist.dir/all] Error 2
```
Adding the following to cmake fixed this problem.
`target_link_libraries(${LF_MAIN_TARGET} PRIVATE m)`

--- a/C/src/sdv/curses.cmake
+++ b/C/src/sdv/curses.cmake
@@ -1,3 +1,3 @@
 find_package(Curses REQUIRED) # Finds the lncurses library
 include_directories(${CURSES_INCLUDE_DIR}) # "The include directories needed to use Curses"
-target_link_libraries(${LF_MAIN_TARGET} PRIVATE ${CURSES_LIBRARIES}) # Links the Curses library
+target_link_libraries(${LF_MAIN_TARGET} PRIVATE ${CURSES_LIBRARIES} PRIVATE m) # Links the Curses library and math

